sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/HexSuite.scala

```diff
@@ -37,4 +37,24 @@ class HexSuite extends SparkFunSuite {
     assert(Hex.hex(Long.MinValue).toString === "8000000000000000")
     assert(Hex.hex(Long.MaxValue).toString === "7FFFFFFFFFFFFFFF")
   }
+
+  test("SPARK-48644: hex bytes values") {
```

Comment:
This test case is irrelevant to SPARK-48644 and we do have test coverage for hex byte arrays

Reply:
@yaooqinn How about submitting a new issue for keeping those test cases?